bin/templates/cordova/lib/build.js

```diff
@@ -409,7 +457,31 @@ function parseOpts(options, resolvedTarget) {
             console.warn('Build option \'' + options[i] + '\' not recognized (ignoring).');
         }
     }
+    if (packageArgs.keystore && packageArgs.alias) {
```

Comment:
Wonding if this should actually not be exclusive with the else if clause. E.g. allow specifying buildConfig, and then also allow setting the password via --password